.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: Build
+
+on:
+  pull_request:
+    branches: ['**']
+  push:
+    branches: ['**']
+    tags: [v*]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        temurin_tag:
```

Comment:
If the `eclipse-temurin` image eventually gets support for ARM64 in its Alpine images, we might be able to reintroduce Alpine support as well.